Project.toml

```diff
@@ -25,7 +25,7 @@ DataInterpolations = "3.1"
 DiffEqBase = "6.45"
 DocStringExtensions = "0.7, 0.8"
 FiniteDifferences = "0.11"
-ModelingToolkit = "4.0.8"
+ModelingToolkit = "4"
```

Comment:
any reason to drop the patch requirement?

Reply:
Just experimenting around and trying to be more flexible wrt the upper bound given MTKs release speed. Possibly `ModelingToolkit = "^4.0.8"`
is the better choice here, but I clicked to fast :smile:

Reply:
it's implicitly ^4.0.8 already.

Reply:
Ah, alright. Thanks :). Than I will remove this PR and just prepare a minor release.